frontend/src/components/UserDetail/DriverDetail.tsx

```diff
@@ -17,23 +33,89 @@ type DriverDetailProps = {
 const DriverDetail = () => {
   const location = useLocation<DriverDetailProps>();
   const driver: DriverDetailProps = location.state;
-  const availToString = (acc: string, [day, timeRange]: string[]) => acc + day + ": " + timeRange + " • ";
-  const parsedAvail = driver.availability.reduce(availToString, "");
+  const availToString = (acc: string, [day, timeRange]: string[]) => acc + day + ': ' + timeRange + ' • ';
+  const parsedAvail = driver.availability.reduce(availToString, '');
   const avail = parsedAvail.substring(0, parsedAvail.length - 2);
-  const vehicle = driver.vehicle ? (driver.vehicle.name + " (" + driver.vehicle.capacity + " people)") : ""
+  const vehicle = driver.vehicle ? (driver.vehicle.name + ' (' + driver.vehicle.capacity + ' people)') : '';
```

Comment:
Try to avoid string concatenations like this, maybe use a template literal. For example, `` `string text ${expression} string text` ``. Also, keep lines from going over 80 characters.

Reply:
My bad. I also meant to select the lines before it including `availToString`